trunk/Editor/NumberedTextBoxUC.Designer.cs

```diff
@@ -1,106 +1,106 @@
-namespace NumberedTextBox

```

Comment:
What's being changed in this file that requires every line to be committed? At a guess I'd assume it was line endings, in which case it shouldn't be committed as part of the case fixes.